igvc_navigation/src/mapper/main.cpp

```diff
@@ -45,105 +35,158 @@ std::tuple<double, double> rotate(double x, double y)
   return (std::make_tuple(newX, newY));
 }
 
-void odom_callback(const nav_msgs::Odometry::ConstPtr &msg)
+void getOdomTransform(const pcl::PointCloud<pcl::PointXYZ>::ConstPtr &msg)
```

Comment:
Can you add doxygen comments? Same with below, all of these functions don't have high level comments explaining what is actually happening.

Reply:
fixed